client/src/components/profile/components/TimeLine.js

```diff
@@ -70,13 +69,12 @@ class Timeline extends Component {
   renderCompletion(completed) {
     const { idToNameMap } = this.props;
     const { id, completedDate } = completed;
-    const challengeDashedName = idToNameMap[id];
+    const challengeTitle = idToNameMap[id].challengeTitle;
```

Comment:
```suggestion
    const { challengeTitle, challengePath } = idToNameMap[id];
```
Just to save a line.